pkgs/applications/audio/bucklespring/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, legacy ? false
+, libinput
+, libudev
+
+, pkg-config
+, makeWrapper
+
+, openal
+, alure
+, libXtst
+, libX11
+}:
+
+with lib;
```

Comment:
Please do not use with lib over the entire scope.

Reply:
i'm not going to add `lib.` in front of every `optionals`, that's ugly

but using it at meta is redundant xD

Reply:
That's how we usually do it in nixpkgs and using with lib over the entire file is way uglier than repeating lib a few times.

Reply:
with should always be used over the smallest scope possible and not over everything.

Reply:
changed to inherit optionals from lib
but did reintroduce `with lib;` at meta